cost/aws/burstable_instance_cloudwatch_credit_utilization/aws_burstable_instance_cloudwatch_credit_utilization.pt

```diff
@@ -0,0 +1,755 @@
+name "AWS Burstable Instance CloudWatch Utilization"
+rs_pt_ver 20180301
+type "policy"
+short_description "Gathers AWS CloudWatch CPU and Burst Credit data for instances on 30 day intervals. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/burstable_instance_cloudwatch_credit_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "low"
+category "Cost"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+parameter "param_num_cpu_surplus_credits" do
+  type "number"
+  label "Number of Surplus Credits to alert on"
+  description "Number of CPU Surplus Credits to report on, Set to -1 to ignore cpu burst credits"
+  default -1
+  min_value -1
```

Comment:
should the default be to ignore.  I think we should use a recommended default, not a default to ignore.

Reply:
They are how much they are billed on, I am not sure what the threshold is there, but let me see what others think.